CMakeLists.txt

```diff
@@ -24,7 +24,7 @@ ENDIF(WIN32 AND NOT UNIX)
 
 
 # Options
-option( RAKNET_ENABLE_SAMPLES 	"Generate RakNet sample projects if true." 		TRUE  )
+option( RAKNET_ENABLE_SAMPLES 	"Generate RakNet sample projects if true." 		FALSE )

```

Comment:
I don't think that changing the default and disable the generation of the sample projects would be good. Hence not going to incorporate this in SLikeNet. Feel free to let me know your rational behind the change so we can reconsider our decision.